src/ui/chat-window.js

```diff
@@ -453,9 +453,8 @@ export class ChatScene extends Phaser.Scene {
         }
 
         this.chatInput = this.input.keyboard.on("keydown", (event) => {
-            // enter
-            if (event.keyCode == 13) {
-                if (this.userMessage.text != "") {
+            if (event.keyCode == Phaser.Input.Keyboard.KeyCodes.ENTER) {
```

Comment:
Object string is pretty long, maybe pre-declare the keycodes so it's cleaner

```js
const KeyCodes = Phaser.Input.Keyboard.KeyCodes;
if (event.KeyCode == KeyCodes.ENTER)
    // ...
```

Reply:
Updated! https://github.com/EthanVieira/rs-clicker/pull/82/commits/fd255f86a83d9b57f1ebb683dada55c54b1f3f83